Cargo.toml

```diff
@@ -78,14 +78,18 @@ features = ["egl", "build_dlls"]
 optional = true
 
 [target.'cfg(target_os = "windows")'.dependencies]
-wio = "0.2"
-winapi = { version = "0.3", features = [
-    "d3d11",
-    "libloaderapi",
-    "winbase",
-    "winerror",
-    "wingdi",
-    "winuser",
+windows = { version = "0.58", features = [
```

Comment:
Could we use windows-sys?

Reply:
From what I can tell, windows-sys does not have the feature set for manipulating DXGI so at best we would be using a mixture of both windows and windows-sys